OOD Tutorials

RobJY opened the pull request:
Added a first pass on the remaining OOD tutorials.  Please let me know if I've missed anything, anything needs to be changed, or if you have any suggestions for how they can be improved.  Thanks!

RobJY commented:
I just ordered them alphabetically, like they are in the Interactive Apps menu, but maybe there's a better way to organize them?

s-sajid-ali commented:
Alphabetical is fine by me.

genericdata commented on docs/hpc/09_ood/02_Alphafold2.mdx:
we're archiving alphafold2 in OOD for torch. We can remove this

genericdata reviewed (changes requested):
Remove the screenshots for the applications drop down. We'll be removing applications shortly when we move to Torch. We'll be obligated to update them, but I think it's just too much work.

RobJY commented:
I removed the interactive apps images as suggested.
Thanks for the suggestions @genericdata !

Any other suggestions @s-sajid-ali and @genericdata ?

RobJY commented:
Cool, thanks for the heads up about the dev url @genericdata !  I've removed those 3 sections and updated Matlab to use Matlab-Proxy.

How's it looking now?  Please let me know if you see anything else that needs to be updated.  Thanks!
